Mail/Views/New Message/Attachments/AttachmentsManager.swift

```diff
@@ -116,8 +116,8 @@ import SwiftUI
         await worker.completeUploadedAttachments()
     }
 
-    func processTextAttachments(_ attachments: [TextAttachable]) async {
-        await worker.processTextAttachments(attachments)
+    func processHtmlAttachments(_ attachments: [HTMLAttachable]) async {
```

Comment:
As the type is called HTMLAttachable, shouldn't the function be called processHTMLAttachments?

Reply:
okay sure, so HTML in caps.